server/base/src/main/java/org/apache/accumulo/server/gc/GcVolumeUtil.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.accumulo.server.gc;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+
+import org.apache.accumulo.core.data.TableId;
+import org.apache.accumulo.core.volume.Volume;
+import org.apache.accumulo.server.ServerConstants;
+import org.apache.accumulo.server.fs.VolumeManager;
+import org.apache.hadoop.fs.Path;
+
+// TODO is this the best place for this code? Code was put in server base because other code besides GC needed to access it...
+
+public class GcVolumeUtil {
+  // TODO normalize using url?
+  private static final String ALL_VOLUMES_PREFIX = "accumulo://allVolumes/";
```

Comment:
I like the consistency of `accumulo://` with other values

Reply:
I would prefer to avoid use of something that we could use later for a data connection string that is user-facing. It could be confusing and unnecessarily ties our future hands.

I still don't like the use of URL-like syntax for something that is explicitly *not* a URL, but if you prefer it for consistency with other things (actual URLs), then how about something like `allVolumes://`?

Reply:
changed to `agcav:/` in 2c3b81d